lib/discourse_sift.rb

```diff
@@ -85,85 +73,28 @@ def self.classify_post(post)
           #Rails.logger.debug("sift_debug:   active flags: #{post.active_flags.inspect}")
 
-          post_action_type = PostActionType.types[:inappropriate]
-          
-          begin
-            PostAction.act(
-              Discourse.system_user,
-              post,
-              post_action_type,
-
-              # TODO: Can't get newline to render by default.  Might need to investigate overriding template or custom template?
-              #message: I18n.t('sift_flag_message') + "</br>\n" + result.topic_string
-              message: I18n.t('sift_flag_message') + result.topic_string,
-            )
-          rescue PostAction::AlreadyActed => e
-          # Post already flagged for this user
-            nil
-          rescue Exception => e
-            Rails.logger.error("sift_debug: Exception when trying flag as system user: #{e.inspect}")
-          end
-          
+          flag_post_as(post, Discourse.system_user, result.topic_string)
 
           # Should we add an extra flags
-          SiteSetting.sift_extra_flag_users.split(",").each { |name|
-            name = name.strip()
-            if !name.blank?
-              begin
-                # send a flag as this user
-                flag_user = User.find_by_username(name)
-                if !flag_user.nil?
-                  PostAction.act(
-                    flag_user,
-                    post,
-                    post_action_type,
-                    message: I18n.t('sift_flag_message') + result.topic_string,
-                  )
-                else
-                  Rails.logger.error("sift_debug: Could not flag post with flag user:#{name}  Could not find user")
-                end
-              rescue PostAction::AlreadyActed => e
-                # Post already flagged for this user
-                nil
-              rescue Exception => e
-                Rails.logger.error("sift_debug: Exception when trying flag extra user: #{e.inspect}")
-              end
-
-            end
-          }
-          
-
-        else
-          # Should post be hidden/deleted until moderation?
-          if !SiteSetting.sift_post_stay_visible
-            # Post Removed Due To Content
-            PostDestroyer.new(Discourse.system_user, post).destroy
-
-            # TODO: Maybe a different message if post sent to mod but still visible?
-            #Notify User
-            if SiteSetting.sift_notify_user
-              SystemMessage.new(post.user).create(
-                'sift_human_moderation',
-                topic_title: post.topic.title
-              )
-            end
-          end
-
-          # Mark Post For Requires Moderation
-          DiscourseSift.move_to_state(post, 'requires_moderation')
-
-          # Trigger an event that community sift has an item for human moderators. This allows moderators to notify chat rooms
-          DiscourseEvent.trigger(:sift_post_failed_policy_guide)
+          SiteSetting.sift_extra_flag_users.split(',').each do |name|
```

Comment:
This feature was added, somewhat as a hack, to support flagging a post by multiple users automatically to get around the `min_flags_stafff_visibility` setting in Discourse. Since that feature is now replaced by score this needs to be removed.